TestProject/Views/HomePageViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  HomePageViewController.swift
+//  TestProject
+//
+//  Created by propelrr-mac-pro on 6/16/22.
+//
+
+import UIKit
+
+class HomePageViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
Use an extension for the `UITableViewDelegate` and `UITableViewDataSource` delegate methods to be more organized.